Userland/Applications/Spreadsheet/SpreadsheetView.cpp

```diff
@@ -127,15 +127,15 @@ void InfinitelyScrollableTableView::mousemove_event(GUI::MouseEvent& event)
             }
         }
 
-        if (m_is_hovering_cut_zone || m_is_dragging_for_copy)
```

Comment:
Commit message: "Rename is_dragging_for_{copy => cut}".

Reply:
Also typo: "Since will be".

Also, this should just be squashed into the commit that changes the behaviour.